client/data/quiz.js

```diff
@@ -1,6 +1,43 @@
 export const data = {
   greeting: 'hello',
   questions: [
+    {
+      question:
+        'Which are correct ways to declare a variable in JavaScript? (strict mode)',
+      answered: false,
+      answers: [
+        {
+          text: 'const name = "Tim";',
+          correct: true,
+          selected: false,
+        },
+        {
+          text: 'name = "Tim";',
+          correct: false,
+          selected: false,
+        },
+        {
+          text: 'let name = "Tim";',
+          correct: true,
+          selected: false,
+        },
+        {
+          text: 'var name = "Tim";',
+          correct: true,
+          selected: false,
+        },
+      ],
+      links: [
+        {
+          text: 'javascript.info',
+          url: 'https://javascript.info/variables',
+        },
+        {
+          text: 'Tyler McGinnis',
+          url: 'https://ui.dev/var-let-const/',
+        },
+      ],
+    },
     {
```

Comment:
Hey @sabanyelkenci I am not sure if this code with questions is necessary, please ask for review our leader @rago89